samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -77,11 +77,15 @@ static struct bt_mesh_sensor_value pres_mot_thres = {
 };
 static struct bt_mesh_sensor_value amb_light_level_ref = ILLUMINANCE_INIT_MILLIS(0);
 static float amb_light_level_gain = 1.0;
-/* Using a dummy ambient light value because we do not have a real ambient light sensor. */
+/* Using dummy sensor values to simulate having a real sensor. */
 static float dummy_ambient_light_value;
+static float dummy_people_count_value;
 
 static bool pres_detect;
-static uint32_t prev_detect;
+static uint32_t prev_pres_detect;
+
+static bool mot_sensed;
```

Comment:
This is supposed to be a percentage. I guess we can keep the sample simple by cycling between 0 and 100% motion, but currently, it's cycling between 0% and 1% motion, masquerading as a boolean, which I find confusing.